packages/next/build/polyfills/fetch/whatwg-fetch.js

```diff
@@ -0,0 +1,10 @@
+/* globals self */
+const fetch = {
+  Headers: self.Headers.bind(self),
+  Request: self.Request.bind(self),
+  Response: self.Response.bind(self),
+  fetch: self.fetch.bind(self),
+}
+
+module.exports = fetch
+module.exports.default = fetch
```

Comment:
Do we need the default variant here? The real package doesn't expose any default. We may even be able to make this whole file ESM exports instead.